aspnetcore/security/blazor/index.md

```diff
@@ -115,6 +115,12 @@ Add a package reference for [Microsoft.AspNetCore.Components.Authorization](http
 
 Implementation of a custom `AuthenticationStateProvider` service for Blazor WebAssembly apps is covered in the following sections.
 
+### Additional claims and tokens from external providers
+
+To store additional claims from external providers, see <xref:security/authentication/social/additional-claims>.
+
+Currently, `GetAuthenticationStateAsync` hasn't been extended to include authentication properties. Therefore, the guidance in the *Save the access token* and *How to add additional custom tokens* sections of the [Additional claims](xref:security/authentication/social/additional-claims#save-the-access-token) topic can't be followed for Blazor apps. Until the framework supports authentication token access in Blazor apps, you can add an API endpoint to the app that returns the tokens for the current user.
```

Comment:
Hold off on this, Javier had some feedback on https://github.com/aspnet/AspNetCore/issues/16831#issuecomment-549941535